### PR TITLE
Dev: add Flare analysis to MUTIS

### PR DESCRIPTION
**Pull request by juanep97**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -12,6 +12,7 @@ A Python package for the analysis of correlations of light curves and their stat
 
    recipes/mutis_workflow.ipynb
    recipes/Synthethic_generation_of_light_curves.ipynb
+   recipes/Flare_analysis.ipynb
```
Comment (Bultako) [inline docs/index.rst:15]: just saying.. in general I'd always use alphabetic ordering in lists, in large lists I've found it useful for folder content comparisons and searching for orphans

File: docs/index.rst [hunk 2]
```diff
@@ -23,4 +24,5 @@ A Python package for the analysis of correlations of light curves and their stat
 
    reference/correlation
    reference/signal
+   reference/flares
```
Comment (Bultako) [inline docs/index.rst:27]: same as above

File: mutis/flares/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Licensed under a 3-clause BSD style license - see LICENSE
+""" Flare analysis """
+
+from .flare import Flare
+from .bayblocks import BayesianBlocks
```
Comment (Bultako) [inline mutis/flares/__init__.py]: alphabetic ordering :)

File: mutis/signal.py [hunk 4]
```diff
@@ -21,16 +21,16 @@
...
-    Parameters
+    Attributes
     ----------
     times : :class:`numpy.ndarray` or :class:`pandas.Series`
         Values of the time axis.
     values : :class:`numpy.ndarray` or :class:`pandas.Series`
         Values of the signal axis.
     fgen : :class:`str`
-        Method used to generate the synthetic signal.
+        Method to generate the synthetic signal.
```
Comment (Bultako) [inline mutis/signal.py:39]: I would add other attributes missing in this list ?

**Review by Bultako — APPROVED** [review]

File: mutis/flares/__init__.py (hunk 3 above)
Comment (juanep97) [reply mutis/flares/__init__.py]: Done.

File: mutis/signal.py (hunk 4 above)
Comment (juanep97) [reply mutis/signal.py:39]: Done.

**Pull request merged**